scripts/combine-and-dedup-fastas.py

```diff
@@ -24,8 +24,10 @@
     with open_file(args.output, "w") as output_handle:
         # Stream sequences from all input files into a single output file,
         # skipping duplicate records (same strain and sequence) and noting
-        # mismatched sequences for the same strain name.
-        for record in read_sequences(*args.input):
+        # mismatched sequences for the same strain name.  In order to
+        # prefer the latter files, we have to reverse the order of the
+        # files.
+        for record in read_sequences(reversed(*args.input)):
```

Comment:
I think you want the star expansion operator to precede the `reversed` call like `read_sequences(*reversed(args.input))`. The current syntax tries to expand a list into `reversed` which doesn't expect multiple arguments and throws a runtime error.

```bash
$ python3 scripts/combine-and-dedup-fastas.py  --input data/example_sequences_aus.fasta.xz data/example_sequences_worldwide.fasta.xz --warn-about-duplicates --output test.fasta
Traceback (most recent call last):
  File "/Users/jlhudd/miniconda3/envs/nextstrain/lib/python3.8/site-packages/augur/io.py", line 33, in open_file
    yield handle
  File "scripts/combine-and-dedup-fastas.py", line 30, in <module>
    for record in read_sequences(reversed(*args.input)):
TypeError: reversed expected 1 argument, got 2
```

Reply:
I was going to say, "i swear this is working in our setup."

And then I realized I did not port [our change out](https://github.com/chanzuckerberg/aspen/blob/trunk/src/backend/aspen/workflows/nextstrain_run/combine-and-dedup-fastas.py.patch) correctly.  #facepalm